server/controllers/QandA/controllers/answers.js

```diff
@@ -0,0 +1,80 @@
+require('dotenv').config();
+const axios = require('axios');
+const api = process.env.API_ENDPOINT;
+
+//import axios from 'axios';
+module.exports = {
+  get: function (req, res) {
+    const question_id = req.body.question_id;
+    const count = req.body.count || 5;
+    const page = req.body.page || 1;
+    axios.get(`${api}qa/questions/${question_id}/answers?`, {
```

Comment:
Might caught conflicts with other teammates. Need to reconcile how we do the URL in the .env file. 

Adding a slash between variable and qa would improve readability.